daffodil-test/src/test/resources/org/apache/daffodil/usertests/SepTests.tdml

```diff
@@ -17,155 +17,305 @@
 -->
 
 <tdml:testSuite
-	xmlns:tdml="http://www.ibm.com/xmlns/dfdl/testData"
-	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xmlns:dfdl="http://www.ogf.org/dfdl/dfdl-1.0/"
-	xmlns:xsd="http://www.w3.org/2001/XMLSchema"
-	xmlns:xs="http://www.w3.org/2001/XMLSchema"
-	xmlns:ex="http://example.com"
-	xmlns:fn="http://www.w3.org/2005/xpath-functions">
-
-	<tdml:defineSchema name="s1" elementFormDefault="unqualified">
```

Comment:
Review with ignore whitespace settings. I reformatted the file due to indentation screw ups. But that changed many lines that aren't part of this change set.